RULES.md

```diff
@@ -0,0 +1,119 @@
+# Based Rules
+
+## Rationale
+
+These rules are implemented to ensure:
+- the repository stays neat and tidy,
+- a consistent Markdown style amongst all recipes,
+- consistent technical details amongst all recipes,
+- appropriate tags and images representative of the recipes on which they are included,
+- recipes are not condescending or verbose, and
+- only authentic, traditional, high-quality food.
+
+Make [based.cooking](https://based.cooking) based again!
+
+
+## Rules
+
+### Recipe format
+
+Model submission files after [example.md](example.md), then put them in `src/`.
+
+### Markdown style
+
+Recipes should start with a title, with a single `#` *on the first line*.
+
+### Technical details
+
+#### File format
+
+- Recipes should start with no empty line at the top and end with no trailing lines.
+- There should be no trailing whitespace.
+- The file needs to be `\n` terminated in Unix-like fashion (if you're using a Unix-like system, it should be automatic).
+- Encode files in UTF-8.
+
+#### File name
+
+- File names should be the name of the dish with words separated by hyphens (`-`), not underscores, and definitely not spaces.
+
+#### Tags
+
+- Recipes should end with a list of relevant tags.
+The syntax is:
+
+```md
+;tags: tag1 tag2 tag3
+```
+
+- The tag line should be a single line, at the end of the markdown file, preceded by a blank line.
+- Add between a few (1-4) tags.
+- **Prioritize existing tags**.
+- As a general guideline, add the country from which the recipe originates if the recipe is representative of said country, using the adjective form (eg. *mexican*, *italian*, etc).
+- Tag the main ingredient if it's something slightly special.
+
+
+##### List of special, categorical tags to use if relevant
+
+- `basic`: for basic recipes that aren't meant to be standalone but are supposed
+  to be incorporated in another recipe.
+- `breakfast`
+- `dessert`
+- `drink`
+- `quick`: for recipes that can be cooked in under ~20 minutes.
+- `side`: side dishes such as mash, fries, etc.
+- `snack`
+- `spread`
+
+#### Images
+
+- Images are stored in `data/pix`.
+- Each recipe can have a title image at the top and perhaps several instructional images as absolutely necessary.
+- Do not add stock images you found on the internet.
+Take a good picture yourself of the actual dish you prepared.
+- If you see a bad or substandard image, you may submit a better one.
+- Images should be in `.webp` format and with as small a file size as possible.
+If you submit an image for say, `chicken-parmesan.md`, it should be added as `pix/chicken-parmesan.webp`.
+- If you would like to add additional directional images, they should be numbered with two digits like `pix/chicken-parmesan-01.webp`, etc.
+
+### Bugmanism is banned
+
+Vegan or vegetarian recipes are fine, but specific bugman food imitating authentic food is disallowed.
```

Comment:
```suggestion
Vegan or vegetarian recipes are fine, but specific bugman food imitating authentic food is disallowed. People can take existing recipes and replace or remove ingredients they don't like with their diet-approved alternative of choice.
```

Reply:
This phrasing implies they should commit the changes they make, whereas it should be clearer that they should make these changes only for themselves in their own kitchens whilst preparing the dish.

Reply:
I made this clear in a new commit.